src/app/shared/pipes/itemLink.ts

```diff
@@ -0,0 +1,12 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { ItemType, typeCategoryOfItem } from '../helpers/item-type';
+import { incompleteItemStringUrl } from '../routing/item-route';
+
+function transform({ id, type }: {id: string, type: ItemType}): any {
```

Comment:
Why to "any"? That's a string and that's a good thing to know (if more piping).

Reply:
Sure, missed

Reply:
fixed